src/main/resources/resource/Intro/Servo01_start.py

```diff
@@ -0,0 +1,49 @@
+#########################################
+# Servo01_start.py
+# categories: intro
+# more info @: http://myrobotlab.org/service/Intro
+#########################################
+# uncomment for virtual hardware
+# Platform.setVirtual(True)
+
+# Every settings like limits / port number / controller are saved after initial use
+# so you can share them between differents script 
+
+# servoPin01 = 4
+
+# port = "/dev/ttyUSB0"
+# port = "COM15"
+
+# create a servo controller and a servo
+arduino = Runtime.start("arduino","Arduino")
+servo01 = Runtime.start("servo01","Servo")
+
+# we tell to the service what is going on
+# intro.isServoActivated = True ## FIXME this gives error readonly
+intro.isServoActivated()
+intro.broadcastState()
+
+# initialize arduino
+# linux or macos -> arduino.connect("/dev/ttyUSB0")
+# print("connecting arduino to serial port")
+# arduino.connect(port)
+
+# set limits
+print("setting min max limits of servo")
+# servo01.setMinMax(0, 180)
+servo01.map(0, 180, 0, 180)
+
+# set rest position
+servo01.setRest(90)
+# set speed
+servo01.setSpeed(100.0)
+
+# attach servo
+# print("attaching servo with pins to controller")
+# servo01.attach(arduino.getName(), servoPin01)
+
+# auto disable - this enables (starts pwm) before a movement
+# and disables (stops pwm) after a movement
+servo01.setAutoDisable(True)
```

Comment:
we should also show an example of  servo01.setIdleTimeout(1000)  to set the amount of time before the auto disable kicks in..

Reply:
Sure, although, this script is not to be seen by the user.
I made this script to be launched via click click during the Intro servo tutorial. Functions to be applied to a servo are mentionned at the end of the tutorial on the Intro webgui page.
setAutoDisable(True) still sets a default IdleTimeout ? correct?

Reply:
default idleTimeout is 3.0 seconds.  yes. setAutoDisable(True) will use the current value of idleTimeout of the servo.